lib/train/transports/vmware.rb

```diff
@@ -0,0 +1,190 @@
+# encoding: utf-8
```

Comment:
There is a lot going on here that I really wish we had integration tests for.  The unit tests are solid - I believe that the accessors and option parsers work.  I'm sure info about integration testing is coming, likely in the resource pack.

Reply:
Yup, I too would like integration tests, I'm just not quite sure how to approach it here.